Mods/ArchipelagoPatches.SC2Mod/Base.SC2Data/Lib6EE41A55.galaxy

```diff
@@ -0,0 +1,33 @@
+include "TriggerLibs/natives"
+
+include "Lib6EE41A55_h"
+
+//--------------------------------------------------------------------------------------------------
+// Library: ArchipelagoPatches
+//--------------------------------------------------------------------------------------------------
+// Custom Script
+//--------------------------------------------------------------------------------------------------
+// Custom Script: Untitled Custom Script 001
+//--------------------------------------------------------------------------------------------------
+void patchMain() {
+}
+
+void lib6EE41A55_InitCustomScript () {
+    patchMain();
+}
+
+//--------------------------------------------------------------------------------------------------
+// Library Initialization
+//--------------------------------------------------------------------------------------------------
+bool lib6EE41A55_InitLib_completed = false;
+
+void lib6EE41A55_InitLib () {
+    if (lib6EE41A55_InitLib_completed) {
+        return;
+    }
+
+    lib6EE41A55_InitLib_completed = true;
+
+    lib6EE41A55_InitCustomScript();
+}

```

Comment:
Ensure that ArchipelagoTriggers will run it on map initialization, otherwise it gets ignored